src/Goerli-DssSpell.t.sol

```diff
@@ -458,14 +502,25 @@ contract DssSpellTest is GoerliDssSpellTestBase {
     }
 
     // RWA tests
+    address RWA007_A_OPERATOR                  = addr.addr("RWA007_A_OPERATOR");
+    address RWA007_A_COINBASE_CUSTODY          = addr.addr("RWA007_A_COINBASE_CUSTODY");
+    
+    RwaLiquidationLike oracle                  = RwaLiquidationLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
+
+    GemAbstract          rwagem_007            = GemAbstract(addr.addr("RWA007"));
+    GemJoinAbstract      rwajoin_007           = GemJoinAbstract(addr.addr("MCD_JOIN_RWA007_A"));
+    RwaUrnLike           rwaurn_007            = RwaUrnLike(addr.addr("RWA007_A_URN"));
+    RwaOutputConduitLike rwaconduitout_007     = RwaOutputConduitLike(addr.addr("RWA007_A_OUTPUT_CONDUIT"));
+    GemAbstract          psmGem                = rwaconduitout_007.gem();
+    RwaInputConduitLike  rwaconduitinurn_007   = RwaInputConduitLike(addr.addr("RWA007_A_INPUT_CONDUIT_URN"));
+    RwaInputConduitLike  rwaconduitinjar_007   = RwaInputConduitLike(addr.addr("RWA007_A_INPUT_CONDUIT_JAR"));
+    uint256 daiPsmGemDiffDecimals              = 10**sub(dai.decimals(), psmGem.decimals());
+
     string OLDDOC = "";
     string NEWDOC = "";
 
     function testDocChange() private { // make public to use
         bytes32 ilk = "RWA009-A";
-        RwaLiquidationLike oracle = RwaLiquidationLike(
-            addr.addr("MIP21_LIQUIDATION_ORACLE")
-        );
```

Comment:
why this was removed?

Reply:
it's already declared

Reply:
hmm, I see what you mean, but this specific test is disabled and not part of the current spell coverage and unless the `oracle` is declared in base tests I don't see why this change is required.

Reply:
true

Reply:
It's duplicate with a storage variable.

Reply:
yeah, in that case, the `oracle` declaration could be moved directly to base tests for future use, together with the interface.